packages/google_maps_flutter/google_maps_flutter/README.md

```diff
@@ -170,3 +170,11 @@ class MapSampleState extends State<MapSample> {
 ```
 
 See the `example` directory for a complete sample app.
+
+## Support
+
+This package is offered via an open source license. It is not governed by the Google Maps Platform Support [Technical Support Services Guidelines](https://cloud.google.com/maps-platform/terms/tssg), the [SLA](https://cloud.google.com/maps-platform/terms/sla), or the [Deprecation Policy](https://cloud.google.com/maps-platform/terms) (however, any Google Maps Platform services used by the library remain subject to the Google Maps Platform Terms of Service).
```

Comment:
> This package is offered via an open source license.

This is true of all of our packages, so not something we would say in every README.